src/ui_events.rs

```diff
@@ -0,0 +1,5 @@
+use slint::SharedString;
+pub enum UIEvent {
+    AddItem(SharedString),
+    // Add more events here as needed
+}
```

Comment:
This could maybe be moved into the app_listview file too? I put it here because it worked, but I think it would make more sense to group all that stuff together...